pkgs/ffigen/CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+## 10.1.0
+
+- Any compiler errors/warnings in source header files will now result in
```

Comment:
What kind of warnings result in wrong generated code? Should we refuse to generate on all warnings?
If we refuse to generate for too many warnings, people might just start using `ignore-source-errors` instead of fixing the issues.

Reply:
True, for now I've only handled warnings/errors from libclang. (Excluding ObjC for now)

But I think anything which can potentially generate invalid bindings, that might silently break at runtime should be covered.